src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1304,7 +1304,10 @@
 
 
 	function m.generateDebugInformation(cfg)
-		m.element("GenerateDebugInformation", nil, tostring(cfg.flags.Symbols ~= nil))
+		local d = cfg.flags.Symbols ~= nil
+		local v
+		if _ACTION >= "vs2015" then v = d and "Debug" or "No" else v = tostring(d) end
```

Comment:
A nitpick: but the coding convention is to put then/else on their own lines, and it does make the code easier to follow.